### ISSUE TITLE
Jio Tv stop working After clicking on channel it do some loading but it not Start 

### ISSUE THREAD

**Issue by ajay7413** [botallen/repository.botallen#332]
**Your Environment**
- Add-on name: [e.g. Jiotv]
- Add-on version: [e.g. 2.0.0]
- Kodi version: [e.g. Kodi 19]
- Operating system version/name: [e.g. Windows 10, LibreElec 9.0, etc...]
- Using from country: [eg. India, USA, etc...]

**Describe the bug**

**To Reproduce**
4. See an error

**Expected behavior**

**Screenshots**

**Additional context**

**Debug log**

The debug log can be found from this link:

**Comment by echakkara** [comment]
I also face this issue since today 30-Dec-22

**Comment by kamelkan** [comment]
yes same issue starting 30 dec 22. sometimes few channel works but stops or freezes the stream in middle

**Comment by KapilMS** [comment]
Also facing the same issue with version V19.3. Also tried installing newer version V20.0 RC2
None of the channels seems to be playing today

**Comment by mk384456** [comment]
i also faced 

> ![Screenshot_2022-12-30-20-29-10-520_org.xbmc.kodi.jpg](https://user-images.githubusercontent.com/121679864/210083933-3452a9ac-2396-4808-b3c4-ac047435ab7c.jpg)![Screenshot_2022-12-30-20-29-04-420_org.xbmc.kodi.jpg](https://user-images.githubusercontent.com/121679864/210083972-d7761314-67a9-4e0e-a1a0-be48e359daf2.jpg)

**Comment by technofreak24** [comment]
Some channels start working after tapping twice or thrice on channel name in FireTV App Remote.
Screenshots and names on https://github.com/botallen/repository.botallen/issues/335#issuecomment-1368438173

**Comment by CSaiteja99** [comment]
is someone working on this problem ? Have you guys heard anything from the admin ?

**Comment by gif123456789** [comment]
Bro facing same problem How to fix this problem?

**Comment by gif123456789** [comment]
i have been using it on my firestick for quite a loong time first time i saw this problem arrived

**Comment by adityapv** [comment]
Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)

**Comment by AMEET82** [comment]
Same problem I am facing.
So many channels are failing to play.
Plz solve.

**Comment by Marlinruno** [comment]
He won't do that, bcs imroot deleted the repository containing in a zip file from his botallen website, so no one can use it, if anybody who had taken a copy if that zip file can use that repository, and or anybody had a backup of that zip file..
